packages/venia-ui/lib/components/CartPage/PriceSummary/giftCardSummary.js

```diff
@@ -71,21 +70,4 @@ const GiftCardSummary = IS_EE
       }
     : () => null;
 
-export const GiftCardSummaryFragment = IS_EE
```

Comment:
The component itself also should no-op if non EE. Is that something we can do with the plugin too?

Reply:
Yes, we can just have an empty export from its CE file and a valid component from its EE file.